app/components/ui/UserWay.jsx

```diff
@@ -0,0 +1,30 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+
+class UserWay extends React.Component {
+  constructor(props) {
+    super(props);
+  }
+
+  componentDidMount() {
+    const { appID } = this.props;
+    const userWayScript = document.createElement('script');
+    userWayScript.type = 'text/javascript';
+    userWayScript.async = true;
+    userWayScript.src = 'https://cdn.userway.org/widget.js';
+    userWayScript.setAttribute("data-account", appID);
+    document.body.appendChild(userWayScript);
+  }
+
+  render() {
+    return (
+      <noscript>Please ensure Javascript is enabled for purposes of <a href="https://userway.org">website accessibility</a></noscript>
+    );
+  }
+};
```

Comment:
Could I suggest that we rewrite this as a function-based component with a hook? Since this mostly just the installation of the script, this can basically be a component that consists of a single `useEffect()` hook along with the return of the `<noscript>` element. I think roughly, this component would look like this with hooks:

```jsx
import React, { useEffect } from 'react';

const UserWay = ({ appID }) => {
  useEffect(() => {
    if (logicCheckingIfAlreadyInstalled) return;

    // The body of your componentDidMount goes here
  });

  return (
      <noscript>Please ensure Javascript is enabled for purposes of <a href="https://userway.org">website accessibility</a></noscript>
  );
}
```

Reply:
I haven't been in React land in a long time, but hooks do look and seem like what the cool kids do. Doing it!

Reply:
Thanks for switching this over to Hooks!